docs/README.md

```diff
@@ -0,0 +1,59 @@
+## Space API Documentation
+
+Contents
+
+- [Usage](#usage)
+- [Class Definitions](class-definitions.md)
+  - [Spaces](class-definitions.md#spaces)
+  - [Space](class-definitions.md#space)
+  - [SpaceMember](class-definitions.md#spacemember)
+
+### Usage
+
```

Comment:
Per the Slack thread, let's move the quickstart to the root readme file and keep the class definitions in the docs folder. It's perhaps worth showing a table of contents in the root readme too similar to what's available at the top of this file.

Reply:
Done